modules/ossm-threescale-webassembly-module-api-object.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assembly:
+//
+// service_mesh/v2x/ossm-threescale-webassembly-module.adoc
+
+[id="ossm-threescale-webassembly-module-api-object_{context}"]
+= The 3scale WebAssembly module `api` object
+
+The `api` top-level string from the 3scale WebAssembly module defines which version of the configuration the module will use.
```

Comment:
Throughout, I see 5 instances of "top-level" with a hyphen, and 2 without. Just a reminder to check if maybe those 2 need to be synced up for consistency.